langlib/langlib-test/src/test/resources/test-src/regexp_negative_test.bal

```diff
@@ -76,3 +77,28 @@ public function testLongIndexFindAllGroups() returns error? {
     string:RegExp r = re `(([0-9A-Z]u)+(ful))`;
     _ = check (trap r.findAllGroups("rubble, trouble, bubble, hubble", 549755813632));
 }
+
+public function testNegativeEmptyCharClass1() returns error? {
+    string pattern = "[]";
+    anydata|error result = trap re `${pattern}`;
+    check assertEqual(result, "Invalid insertion in regular expression: Empty character class disallowed");
+}
+
+public function testNegativeEmptyCharClass2() returns error? {
+    anydata|error result = trap re `(abc${"[]"})`;
+    check assertEqual(result, "Invalid insertion in regular expression: Empty character class disallowed");
+}
+
+public function testNegativeEmptyCharClass3() returns error? {
+    anydata|error result = trap regexp:fromString("(([abc])|([]))");
+    check assertEqual(result, "Failed to parse regular expression: Empty character class disallowed");
+}
+
+function assertEqual(anydata|error actual, anydata|error expected) returns error? {
```

Comment:
This function is more specialised for checking error messages ryt?
Shall we update the function name to something like `assertEqualError Messages`.
wdyt?